src/main.jsx

```diff
@@ -4,17 +4,55 @@ import './styles/main'
 
 import React from 'react'
 import { render } from 'react-dom'
+import { Router, Route, Redirect, hashHistory } from 'react-router'
 import { I18n } from './plugins/preact-polyglot'
 
-import App from './components/app'
+import App from './components/App'
+import Table from './components/Table'
 
 const context = window.context
 const lang = document.documentElement.getAttribute('lang') || 'en'
 
 document.addEventListener('DOMContentLoaded', () => {
   render((
     <I18n context={context} lang={lang}>
-      <App />
+      <Router history={hashHistory}>
+        <Route component={(props) =>
+          <App {...props} />}
+        >
+          <Redirect from='/' to='files' />
+          <Route
+            path='files'
+            component={(props) =>
+              <Table {...props} />
```

Comment:
Hmm… I understand but I'm a little bit confuse about having the file list loaded in the _main.jsx_ file. Is it because I think it has more it's place in a store that should be called from the `Table` component itself? I can't really what anoy me in the way… @goldoraf, If you have some suggestion about the best way to do this 😃